pkg/apis/componentconfig/edgecore/v1alpha2/default_kubelet_configuration.go

```diff
@@ -0,0 +1,99 @@
+package v1alpha2
+
+import (
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	componentbaseconfigv1alpha1 "k8s.io/component-base/config/v1alpha1"
+	kubeletconfigv1beta1 "k8s.io/kubelet/config/v1beta1"
+	"k8s.io/kubernetes/pkg/kubelet/qos"
+	kubetypes "k8s.io/kubernetes/pkg/kubelet/types"
+	utilpointer "k8s.io/utils/pointer"
+
+	"github.com/kubeedge/kubeedge/common/constants"
+)
+
+const (
```

Comment:
Why not just import these constants from k8s code?

Reply:
> Why not just import these constants from k8s code?

fixed.